demisto_sdk/__main__.py

```diff
@@ -152,9 +153,24 @@ def get_context_arg(args):
     @functools.wraps(func)
     def wrapper(*args, **kwargs):
         logging_setup(
-            console_log_threshold=kwargs.get("console_log_threshold") or logging.INFO,
-            file_log_threshold=kwargs.get("file_log_threshold") or logging.DEBUG,
-            log_file_path=kwargs.get("log_file_path") or None,
+            console_log_threshold=get_config_param_kwargs_env(
+                kwargs,
+                "console_log_threshold",
+                "DEMISTO_SDK_CONSOLE_LOG_THRESHOLD",
+                logging.INFO,
+            ),
+            file_log_threshold=get_config_param_kwargs_env(
+                kwargs,
+                "file_log_threshold",
+                "DEMISTO_SDK_FILE_LOG_THRESHOLD",
+                logging.DEBUG,
+            ),
+            log_file_path=get_config_param_kwargs_env(
+                kwargs,
+                "log_file_path",
+                "DEMISTO_SDK_LOG_FILE_PATH",
+                None,
+            ),
```

Comment:
this is repeated thrice, Can we use a method instead? I don't mind the method being called 3 times instead.